src/scry/protocol/message_type.cr

```diff
@@ -0,0 +1,8 @@
+module Scry
```

Comment:
Should be `Scry::Protocol` ?

I just realized that (all?) the LSP protocol is not in a `Protocol` module, as I think it should, to properly see on usage when you're using a protocol type or not (that's important!).

This can be done later in another PR though!

Reply:
Yeah, I'm already working on that :+1: